core/gotemplate/Readme.md

```diff
@@ -1,60 +1,149 @@
 # Gotemplate
 
-Flamingo comes with a wrapped `html/template` as simple default template engine.
-
-Refer to [golang.org/pkg/html/template/](https://golang.org/pkg/html/template/) for the basic documentation
+Flamingo comes with a wrapped `html/template` as a simple default template engine.
 
 ## Structured templating
 
 ### Template directory
 
-This module allows to set up a deeply nested directory structure with template (html) files. 
+This module allows you to set up a deeply nested directory structure containing template files with the `.html` type ending. 
 These files can be referenced from a controller by just using the path without `.html`.
 
-For example, to render `deep/nested/index.html` in your controller, just call
+An example for such a directory structure could be:
+
+```text
+/project/
+  /templates/
+    /deep/
+      /nested/
+        /index.html
+  main.go
+  go.mod
+```
+
+Here `index.html` resembles an example template. For our purposes, it contains the following content:
+
+```html
+<!-- /templates/deep/nested/index.html -->
+<!doctype html>
+<html>
+  <head>
+      <meta charset="utf-8">
+      <title>Hello World</title>
+  </head>
+  <body>
+    <main>
+      <h1>Huzzah! It works!</h1>
+      <p>
+        This is an example text.
+      </p>
+    </main>
+  </body>
+</html>
+```
+
+You can refer to the [html/template documentation](golang.org/pkg/html/template/) for further information on how to fill your template.
+
+To render `index.html` in your controller, just call
+
+```go
+return controller.responder.Render("templates/deep/nested/index")
```

Comment:
```suggestion
return controller.responder.Render("deep/nested/index")
```

The directory name is set by config `core.gotemplate.engine.templates.basepath` which defaults to `templates`